internal/io/event.go

```diff
@@ -1,28 +1,26 @@
 //
-// Copyright (C) 2021 IOTech Ltd
+// Copyright (C) 2021-2022 IOTech Ltd
 //
 // SPDX-License-Identifier: Apache-2.0
 
 package io
 
 import (
-	"io"
+	"fmt"
 
 	"github.com/edgexfoundry/go-mod-core-contracts/v2/errors"
 )
 
 // To avoid large data causing unexpected memory exhaustion when decoding CBOR payload, defaultMaxEventSize was introduced as
 // a reasonable limit appropriate for handling CBOR payload in edgex-go.  More details could be found at
 // https://github.com/edgexfoundry/edgex-go/issues/2439
```

Comment:
remove this comment, because this function is not for CBOR only.

Reply:
removed